test/common/index.js

```diff
@@ -246,7 +246,7 @@ Object.defineProperty(exports, 'opensslCli', {get: function() {
 
 Object.defineProperty(exports, 'hasCrypto', {
   get: function() {
-    return process.versions.openssl ? true : false;
+    return Boolean(process.versions.openssl);
```

Comment:
No strong opinion from me. I think it used to be preferred in hot paths for performance. No idea if that's still the case. Regardless, it's totally sensible to value readable code over performant code in `test/common/index.js`.